scripts/ci.js

```diff
@@ -22,6 +25,74 @@ const execaOpts = {
   }
 };
 
+const isHeadCommitOfPR = async () => {
+  const prNumber = process.env.CODEBUILD_SOURCE_VERSION.replace('pr/', '');
+
+  const { stdout } = await execa('curl', [
+    '-H',
+    '"Authorization:',
+    'token',
+    `${process.env.GITHUB_TOKEN}"`,
```

Comment:
NIT: Would this be better as:

```js
  const { stdout } = await execa('curl', [
    '-H',
    `"Authorization: token ${process.env.GITHUB_TOKEN}"`,
    `https://api.github.com/repos/FormidableLabs/badges/pulls/${prNumber}/commits`
  ]);
```

?

Reply:
Better yet, use the SDK (`@octokit/rest`) we already depend on!

```js
octokit.pulls.listCommits({
  owner,
  repo,
  pull_number
})
```